### DISCUSSION TITLE
LaTeX in figure captions with lightbox

### DISCUSSION THREAD

**Discussion by snhansen** [quarto-dev/quarto-cli#7591] [Q&A] [answered]
### Description

I noticed that LaTeX syntax isn't rendered properly in the lightbox view of figures. Is that a bug or should I use some other syntax?

	---
	format: 
	  html:
	    lightbox: true
	---

	```{r}
	#| fig-cap: "$\\beta_0$"
	#| label: fig-label
	library(ggplot2)
	ggplot(data = mtcars, aes(y = mpg, x = wt)) + geom_point()
	```

**Comment by cscheid** [comment]
@dragonstyle How can we tell lightbox to send the captions through our math processing?

**Reply by dragonstyle** [reply]
We'll need to explicitly typeset the math on the client when the Lightbox appears (we do some of this for hover xrefs, but it isn't really generalized yet and definitely doesn't address Lightbox elements at all). It should be pretty straightforward, but is definitely a bug. I'll open one based upon this thread and we'll try to fix it soon!

**Comment by mcanouil** [comment, marked as answer]
Closing in favour of:
- https://github.com/quarto-dev/quarto-cli/issues/7592
